published/004-2016-Jan-25.md

```diff
@@ -0,0 +1,92 @@
+# IPFS Weekly #4
+
+[IPFS](//ipfs.io/) is a new hypermedia distribution protocol, addressed by content and identities, aiming to make the web faster, safer, and more open. In these posts, we highlight some of the development that has happened in the past week. For anyone looking to get involved, follow the embedded hyperlinks, search the wealth of information on [GitHub](//github.com/ipfs) or join us on [IRC](//webchat.freenode.net/?channels=ipfs) (#ipfs on the Freenode network).
+
+If you would like to get this update as an email, sign up for our [weekly newsletter](//tinyletter.com/ipfsweekly)!
+
+Here are some of the highlights for the [January 25th](//github.com/ipfs/pm/issues/84) sprint:
+
+## Updates
+
+* [**js-ipfs**](//github.com/ipfs/js-ipfs): [@diasdavid](//github.com/diasdavid) created [js-ipfs-merkle-dag](//github.com/vijayee/js-ipfs-merkle-dag) and [js-ipfs-blocks](//github.com/ipfs/js-ipfs-blocks), which will help IPFS deal with [MerkleDAGs](https://github.com/ipfs/specs) and are extensible enough to allow others to build their own MerkleDAG structures. Also, js-ipfs is now able to import a file into the MerkleDAG, an important milestone. An architecture graph has been spec'ed out [here](https://github.com/ipfs/js-ipfs#ipfs-core-implementation-architecture) to help newcomers understand how js-ipfs works. Finally, lots of new issues and low-hanging fruit have been tagged for [contributors to tackle](//github.com/ipfs/js-ipfs/issues).
```

Comment:
Don't forget to tribute the merkle-dag also to https://github.com/vijayee, he is totally awesome :)